lib/utils.js

```diff
@@ -150,6 +171,8 @@ module.exports = {
         return currentExports;
       }, {});
 
+    const exportNodes = exportNodesESM || exportNodesCJS;
```

Comment:
when `ast.sourceType === "module"`, it seems no need to calc `exportNodesCJS`?

what do you think checking its value like: `exportNodes = ast.sourceType === "module" ? calcEsmExport() : calcCjsExport();`?

Reply:
Will update.

Reply:
Updated.